src/concrete/ml/search_parameters/p_error_search.py

```diff
@@ -104,6 +110,16 @@ def compile_and_simulated_fhe_inference(
     """
 
     compile_params: Dict = {}
+
+    default_configuration = Configuration(
+        dump_artifacts_on_unexpected_failures=False,
+        enable_unsafe_features=True,
+        use_insecure_key_cache=True,
+        insecure_key_cache_location="ConcreteNumpyKeyCache",
+        parameter_selection_strategy=ParameterSelectionStrategy.MONO
```

Comment:
we can merge this way and wait for the new CP that fixes this but we should add a TODO comment and an issue

Reply:
I created an issue to remove the mono execution: https://github.com/zama-ai/concrete-ml-internal/issues/3978
I also changed the default_configuration in the conftest.py, to make it more clear.

Reply:
thanks for the issue (maybe could you explain a bit more what we would need to do in order to fix the issue ? I mean in the issue), as Andrei said could you put a FIXME here so that we know that there's actually an issue here ?

Reply:
also I think this change is not necessary (at least for the strategy part) if you directly force KNN to MONO strategy in their compilation step as done for linear models

Reply:
Following your advice, I forced the KNN to run in MONO strategy